diff --git a/fbs/_defaults/src/installer/windows/Installer.nsi b/fbs/_defaults/src/installer/windows/Installer.nsi
old mode 100644
new mode 100755
index 47dfb0f..a18ea8c
--- a/fbs/_defaults/src/installer/windows/Installer.nsi
+++ b/fbs/_defaults/src/installer/windows/Installer.nsi
@@ -49,7 +49,7 @@ FunctionEnd
     !define MUI_FINISHPAGE_RUN
     !define MUI_FINISHPAGE_RUN_CHECKED
     !define MUI_FINISHPAGE_RUN_TEXT "Run ${app_name}"
-    !define MUI_FINISHPAGE_RUN_FUNCTION "LaunchLink"
+    !define MUI_FINISHPAGE_RUN_FUNCTION "LaunchAsNonAdmin"
   !insertmacro MUI_PAGE_FINISH
 
   !insertmacro MUI_UNPAGE_CONFIRM
@@ -95,7 +95,7 @@ Section "Uninstall"
 
 SectionEnd
 
-Function LaunchLink
-  !addplugindir "."
-  ShellExecAsUser::ShellExecAsUser "open" "$SMPROGRAMS\${app_name}.lnk"
-FunctionEnd
\ No newline at end of file
+Function LaunchAsNonAdmin
+  Exec '"$WINDIR\explorer.exe" "$InstDir\${app_name}.exe"'
+FunctionEnd
+
